runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/stream/HttpClientFactory.java

```diff
@@ -112,18 +160,22 @@ public final class HttpClientFactory implements HttpStreamFactory
                     .build();
 
     private static final String8FW HEADER_AUTHORITY = new String8FW(":authority");
+    private static final String8FW USER_AGENT = new String8FW("user-agent");
     private static final String8FW HEADER_CONNECTION = new String8FW("connection");
     private static final String8FW HEADER_CONTENT_LENGTH = new String8FW("content-length");
     private static final String8FW HEADER_METHOD = new String8FW(":method");
     private static final String8FW HEADER_PATH = new String8FW(":path");
     private static final String8FW HEADER_STATUS = new String8FW(":status");
     private static final String8FW HEADER_TRANSFER_ENCODING = new String8FW("transfer-encoding");
     private static final String8FW HEADER_UPGRADE = new String8FW("upgrade");
+    private static final String8FW HEADER_HTTP2_SETTINGS = new String8FW("HTTP2-Settings");
 
     private static final String16FW METHOD_GET = new String16FW("GET");
     private static final String16FW PATH_SLASH = new String16FW("/");
-    private static final String16FW RETRY_AFTER_0 = new String16FW("0");
     private static final String16FW STATUS_101 = new String16FW("101");
+    private static final String16FW H2C = new String16FW("h2c");
+    private static final String16FW H2_UPGRADE = new String16FW("Upgrade, HTTP2-Settings");
```

Comment:
The naming convention for these constants is `<HEADER_NAME>_<HEADER_VALUE>`, such as `METHOD_GET` or `PATH_SLASH`.

Following the same naming convention gives `UPGRADE_H2C` and `CONNECTION_UPGRADE_HTTP2_SETTINGS`.